crates/loro-internal/src/handler.rs

```diff
@@ -311,50 +899,122 @@ impl ValueOrHandler {
         state: &Weak<Mutex<DocState>>,
     ) -> Self {
         if let LoroValue::Container(c) = value {
-            ValueOrHandler::Handler(Handler::new(c, arena.clone(), txn.clone(), state.clone()))
+            ValueOrHandler::Handler(Handler::new_attached(
+                c,
+                arena.clone(),
+                txn.clone(),
+                state.clone(),
+            ))
         } else {
             ValueOrHandler::Value(value)
         }
     }
+
+    fn to_value(&self) -> LoroValue {
+        match self {
+            Self::Value(v) => v.clone(),
+            Self::Handler(h) => {
+                LoroValue::Container(ContainerID::new_normal(ID::NONE_ID, h.c_type()))
```

Comment:
Why use `NONE_ID`? Is it always `NONE_ID`?

Reply:
Nice catch